### DISCUSSION TITLE
An electron and positron combined as a dark matter particle?

### DISCUSSION THREAD

**Discussion by mikehelland** [a-cosmology-group/acg#170] [Astrophysics and Physics]
### Preface

When it comes to dark matter there is:

1. its role in the rotation of galaxies (astrophysics)
2. its role in the universe's beginning, and evolution (cosmology)

As far as this post goes, I pretty much just have the first one in mind, its role in the "extra mass" galaxies seemingly require. 

By posting this idea, let me just say I'm not ruling out MOND. I'm not taking the position that there is dark matter and it is some kind of unknown particle. 

I also don't have this worked out in any detail beyond what I'll share here. This is just a conjecture. 

So, assume a DM particle exists, it has neutral charge, and is non-baryonic, meaning its not made of protons or neutrons or quark made stuff like that. It does have mass. 

### Conjecture

When an electron and a positron meet they are thought to annihilate. They let off light that contains as much energy as their mass, so their mass just disappears. Also, their charge disappears, and so would any kinetic energy they had right before they annihilated. 

What if the energy let off from this meeting is actually kinetic energy being converted? 
Or maybe the charge, or electric potential energy being converted? 

The electron and positron actually remain at a distance of zero from each other, becoming one perfectly neutral, non-baryonic particle with a mass equal to two electrons. 

When we observe an electron and positron being created by a cosmic ray, according to this conjecture, what we're actually observing is a cosmic ray breaking an  electron and positron pair apart.

When the pair is "created", why don't they simply annihilate right away? They should be drawn to each other, but we observe them headed in different directions, usually annihilating with some other random particles. It would seem not only do the particles and their masses and their charges get created out of nothing, but they also have the kinetic energy to overcome the force that attracted them together.
 
This conjecture is that the cosmic ray imparts enough energy on the electron/position pair to either give them the kinetic energy to get away from each other, or at least "neutralize" the effect they have on each other so they can get away.

Such a particle would be fairly light, relative to the rest of stuff in the galaxy, so while the rest of the stuff kind of sinks to the center the electron/positron pairs float to the top. This would explain why they form a "halo", they're like cloud cover over a planet.

### Test

Since we already know how to detect it (you can build your own detector for like $30 bucks), it should be somewhat feasible to test.

Send some dry ice in a fish tank with a web cam on it to different places around the solar system. If we suspect dark energy should be more abundant in some places and less abundant than others, we should see more "pair creation" where there is more dark matter.

### Missing anti-matter?

If a galaxy was actually considerably more dark matter than ordinary matter, that means that half of all dark matter is anti-matter. So the question of why there is so little anti-matter around might be significantly alleviated. There is a lot more around than meets the eye. It is around. It's just locked up in dark matter.

**Comment by Francois-Zinserling** [comment]
I like the way you are thinking but
Disclaimer: I think both CDM and MOND are incorrect. CDM because it is based on incorrect assumptions, and MOND because their deviation from 1/r^2 is ad-hoc. Either of these work for many instances but do not offer a holistic solution.

Onto the positron-electron pair
- the photons released after annihilation total the energy of the mass AND the kinetic energy. With 'zero' kinetic energy, 2 photons are released totalling exactly the total $mc^2$ of the original masses, and if there were significant kinetic energy, 3 or more photons are released. (I think always >2, but sometimes not detectable). So all the energy, including mass whizzes off as photons. This experiment (in my opinion) is the most definitive proof of $E=mc^2$

- The reverse is also true. Pair production requires energy $E=mc^2$ for a probability to 'recreate' the 2 masses

- Yes, they would immendiatley want to annihilate, and most of the time they do. 
However in a very strong magnetic field, if they were moving parallel at a significant velocity, it would almost appear 
as if the + and - repel each other. Once you have them far enough away from each other, you can bind e.g. the electron into an atom and you have a free positron. (until it finds another electron and annihilates)
- There are other ways to (slightly) delay annihilation, e.g. by changing spin of one of the particles, and/or by a huge kinetic energy difference.

**Reply by sahil5d** [reply]
The MOND (or perhaps better named Milgrommatic gravity or M gravity) radial acceleration relation (RAR) is not ad-hoc as in "extraneous addition to prop up a failing status quo theory". What it is is a new model of gravity *competing* with the failing status quo.

Milgrom's work should be heralded as among the best scientific research ever because it literally is making observations, modeling data extremely carefully and economically, and making predictions about future observations that actually turn out true (like successfully predicting the rotation data of 100s of galaxies not in the original dataset)! All the put-downs of MOND make no sense. It actually is among the best scientific research ever conducted.

If you are looking for a holistic solution or explanation for the radial acceleration relation, then one avenue to explore is whether all the mass exterior to a galaxy's volume (the rest of the mass of the universe, if shaped like a hypersphere with radius 13.8 Gly) would have attractive force that would account for the ~ `sqrt(GM/r^2 * c^2 / (87 Gly))` function.

**Reply by mikehelland** [reply]
Thank you for the response, @Francois-Zinserling !

I'm not particularly (no pun intended) attached to the conjecture, although there are a couple loose ends that I see.

1. How fast is are the particles moving in an absurdly small measure (eg, the last attosecond, or last Planck length) before they annihilate? If I remember right, if they are moving around 80% the speed of light their total energy would be the rest mass (2 electron masses) doubled (so 4 electron masses). This means you could radiate away 2 electron masses worth of energy, and still have 2 electrons masses left.

2. Every charged particle in the universe has some effect on every other charged particle. Wouldn't its absence be missed somehow? Wouldn't that need to be balanced out somehow if it just disappeared?

**Reply by Francois-Zinserling** [reply]
@mikehelland 
1. Yes, even in a short distance the particles can reach absurd velocities, because the Coulomb force is so much stronger than e.g. gravity. Still, if their total energies equal $E_m+E_k=γmc^2$ (where γ>1 is the Lorentz factor) the sum total of the photon energies will include mass energy AND kinetic energy. E.g. positron and electron annihilate (mass ~0.5 MeV each) at net kinetic energy between them e.g. 2MeV, then total energy = 0.5+0.5+2 = 3MeV and you will find all of that in the photons after annihilation.
2. Now you sound like a string theorist. Just kidding. We observe the universe appears to be charge neutral but +/- can be recreated (not so easily) as they are annihilated. There is a move that says baryon numbers are conserved (e.g. protons) but then we have probably blown up enough of them in the LHC to refute this. As long as a $+$ and $-$ particles are created or disappear together, no laws are violated. If one $-$ particle should disappear on its own, the universe will be broken.

**Reply by Francois-Zinserling** [reply]
@sahil5d the physics community's biggest concern with MOND is that it is not relativistic. Just as Newton isn't. 

That is not my biggest concern. My concern is that they modified the Newton equation to fit the data, and then added some conditions to when it should apply, and when it shouldn't. It is not a new theory, just a hack of Newton, and provides no explanation as to why the gravity should be different under e.g. low acceleration scenarios. The results have been mostly very good, because of course they pick their results, but there have been some spectacular failures. There should have been no failures. If Newton gets rejected because it got Mercury's precession wrong to 0.001% then MOND should have been out the door long ago.

If MOND had come along and said observations show that $x+y$ needs to be added to Newton as e.g. velocity or acceleration components I would still have been ok with it, because there must be components like those. But they messed with the $1/r^2$ component which is just plain sacrilege! :)

In my mind the $1/r^2$ is a simple result of a field entering or exiting a Gaussian sphere. You can change the spere to an ellipsoid and get a different equation, but then you need to change your entire coordinate system too. 

The reason why MOND is getting so much airtime is because we are all hoping for something better to arrive. (GR is not the final answer either).

**Reply by sahil5d** [reply]
> My concern is that they modified the Newton equation to fit the data, and then added some conditions to when it should apply, and when it shouldn't. It is not a new theory, just a hack of Newton, and provides no explanation as to why the gravity should be different under e.g. low acceleration scenarios.

I think your concern here is not scientific, not constructive, and not accurate. Milgrommatic gravity is the best description of galactic rotation so far. It's better than a classic 1/r^2. It's better than 2-free-parameter-per-galaxy dark matter.

The ultimate model of gravity will have a radial acceleration relation that will either be or resemble the Milgrommatic radial acceleration relation, therefore dismissing the Milgrommatic radial acceleration is illogical.

The low-acceleration scenario condition is scaffolding. That is good science. Also, there is one idea in progress to use a function like
`a = GM/R^2 + GM/R^2 / (e^sqrt(GM/R^2 / (c^2/(2pi * 13.8Gly))) - 1)` which doesn't have conditions.

I suggested an explanation in the previous reply. The mass exterior to a galaxy's volume. For example, here's a simpler model: Consider a 1D universe bent into a circle, with say millions and millions of point masses distributed as a circle. And force between pointmasses Gmm/r^2. What is the net effect of the situation of the 1D universe curving into a circle? There's a net pull to the center on each pointmass, but let's see, how does the pointmass stay on the circle and not move into the circle area toward the center? The pointmass must have a higher velocity around the circle, and that might be a situation that can maintain equilibrium of being on the circle.?

> The results have been mostly very good, because of course they pick their results, but there have been some spectacular failures. There should have been no failures.

What do you mean they "pick their results"? The Milgrommatic function works on 100s of galaxies outside the original dataset, and if you were to gather data of 100s of more galaxies, I would bet that the function makes successful predictions for > 80% of them. Its "failures" are overblown, like the Bullet Cluster, which is an extreme collision situation in the first place. Big Bang LCDM "explains" the bullet cluster with a bunch more free parameters, so that's no explanation either.

> If Newton gets rejected because it got Mercury's precession wrong to 0.001% then MOND should have been out the door long ago.

ACG should spend its energy evolving Milgrom's theory. Not taking cheap shots.

**Reply by bgreenman** [reply]
We are in the early stages of developing a new theoretical model of radial acceleration of mass at galaxy edges. MOND or new theories in that particular subject are just beginning to be sorted out
